### PR TITLE
docs: add java tls docs

### PR DESCRIPTION
**Pull request by sunng87**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The GreptimeDB Java ingester library utilizes gRPC for writing data to the datab
 To connect to GreptimeCloud, using information below:
 
 - Host: `<host>`
-- Port: `4001`
+- Port: `5001`
```
[the remark this answers is not in the document]
Comment (sunng87) [reply docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md:8]: Good catch. We use new port for secure connection. Let me update the one in code.

[the remark this answers is not in the document]
Comment (tisonkun) [reply docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md:8]: However, secure connection is an advanced topic? Or we'd always use this port without extra setup.

[the remark this answers is not in the document]
Comment (tisonkun) [reply docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md:8]: I just found that it's on cloud. But IIRC cloud default is 443?

[the remark this answers is not in the document]
Comment (sunng87) [reply docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md:8]: Here we use different port for grpc(5001) and http (443). On the cloud, secure transport is enabled by default and we will sunset plaintext port(4001) once our SDKs and vector are ready for TLS.

File: docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md
Comment (tisonkun) [inline docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md:1]: https://docs.greptime.cn/greptimecloud/integrations/sdk-libraries/java This page has a Chinese companion IMO. We should update for that also.

Comment (sunng87) [reply docs/v0.7/en/greptimecloud/integrations/sdk-libraries/java.md:1]: Yes, just updated the translated version.

**Review by tisonkun — APPROVED** [review]

**Review by nicecui — APPROVED** [review]

**Pull request merged**